app/views/devise/sessions/new.html.erb

```diff
@@ -1,22 +1,22 @@
-<div class ="main">
-  <div class="container">
-    <%= render_svg 'Mobile-Logo' %>
-    <h1 class="header">Log in</h1>
+<div class ="login">
```

Comment:
If this PR is only to add the Footer, you will need to remove these login / register changes, so your PR will look cleaner, what do you think?

Reply:
We added those changes as hotfix, we didn't think it should need its own PR

Reply:
yes, you need to create a hotfix PR for that, and also, don't mark these conversations if are not really resolved